examples/specs/normalized/box-plot_minmax_1D_horizontal_normalized.vl.json

```diff
@@ -44,14 +12,16 @@
       },
       "encoding": {
         "x": {
-          "field": "lowerWhisker",
-          "type": "quantitative",
           "axis": {
             "title": "population"
-          }
+          },
```

Comment:
Hmm, weird. Does it actually revert it or does the aggregation not happen in the encoding? What line chauses this change anyway?